src/NLog/LayoutRenderers/ExceptionLayoutRenderer.cs

```diff
@@ -309,15 +305,27 @@ private void AppendInnerException(Exception currentException, StringBuilder buil
             builder.Append(InnerExceptionSeparator);
             AppendException(currentException, InnerFormats ?? Formats, builder);
         }
-
+#if !NET3_5 && !SILVERLIGHT4
+        private void AppendException(Exception currentException, List<ExceptionRenderingFormat> renderFormats, StringBuilder builder, AggregateException aggregateException = null)
```

Comment:
If you change it to `Exception aggregateException = null`, then you can remove `#if !NET3_5 && !SILVERLIGHT4`

Reply:
i think i'll keep the `#if` and input param as type AggregateException as it only available post NET3_5

Reply:
I prefer no `#if` if possible. As that is hard with refactoring. 👼 (no refactoring tool will handle those fully correct when moving)